### ISSUE TITLE
Synthetic Data-based Detection of Zebras in Drone Imagery. (arXiv:2305.00432v1 [cs.CV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#25853]
Synthetic Data-based Detection of Zebras in Drone Imagery. (arXiv:2305.00432v1 [cs.CV])<br>
https://ift.tt/6NGkcDW<br>
<p>Datasets that allow the training of common objects or human detectors are
widely available. These come in the form of labelled real-world images and
require either a significant amount of human effort, with a high probability of
errors such as missing labels, or very constrained scenarios, e.g. VICON
systems. Likewise, uncommon scenarios, like aerial views, animals, like wild
zebras, or difficult-to-obtain information as human shapes, are hardly
available. To overcome this, usage of synthetic data generation with realistic
rendering technologies has recently gained traction and advanced tasks like
target tracking and human pose estimation. However, subjects such as wild
animals are still usually not well represented in such datasets. In this work,
we first show that a pre-trained YOLO detector can not identify zebras in real
images recorded from aerial viewpoints. To solve this, we present an approach
for training an animal detector using only synthetic data. We start by
generating a novel synthetic zebra dataset using GRADE, a state-of-the-art
framework for data generation. The dataset includes RGB, depth, skeletal joint
locations, pose, shape and instance segmentations for each subject. We use this
to train a YOLO detector from scratch. Through extensive evaluations of our
model with real-world data from i) limited datasets available on the internet
and ii) a new one collected and manually labelled by us, we show that we can
detect zebras by using only synthetic data during training. The code, results,
trained models, and both the generated and training data are provided as
open-source at https://ift.tt/3odSH7X.
</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
